core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -323,7 +323,12 @@ public void onLoad(ItemGroup<? extends Item> parent, String name) throws IOExcep
         builds = buildMixIn.getRunMap();
         triggers().setOwner(this);
         for (Trigger t : triggers()) {
-            t.start(this, Items.currentlyUpdatingByXml());
+            try {
+                t.start(this, Items.currentlyUpdatingByXml());
+            } catch (Exception e) {
+                Logger.getLogger(AbstractProject.class.getName()).log(Level.WARNING,
+                        "could not start trigger '" + t + "' while loading project '" + name + "'", e);
```

Comment:
Well I am thinking of reading through `support.zip` log files with some messages written in German, which slows me down. :-) The class name is what you actually need to know in order to fix the problem anyway.